src/services/fileStorage/model.js

```diff
@@ -1,63 +1,61 @@
-'use strict';
-
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
-const permissionTypes = ['can-read', 'can-write'];
+const permissionSchema = new Schema({
+	refId: {
+		type: Schema.Types.ObjectId,
+		refPath: 'refPermModel'
+	},
+	refPermModel: {
+		type: String,
+		enum: ['user', 'role']
+	},
+	write: { type: Boolean, default: true },
```

Comment:
Following the logic that is used in our stack, we have use permission as array of strings. In this way ['can-read','can-write',....]
I need also the model as fast as possible in n21/main. Please create a stable version, solve the conflicts and make a merge in n21/main.

Reply:
Using an array of constant string is really error prone. You can easily write `canread` instead and wonder why things do not work. Additionally you always need to check if there is already a `can-read/write` in the array to avoid duplicates.

Reply:
ok then we go this way